locksmith/src/controllers/userController.ts

```diff
@@ -125,6 +128,17 @@ namespace UserController {
     let result = await UserOperations.getCards(emailAddress)
     return res.json(result)
   }
+
+  export const keys = async (req: Request, res: Response) => {
+    if (env == 'development') {
```

Comment:
Why do we not support this on dev?

Reply:
At the moment, the feature is supported by external APIs (with support on mainnet(1) & rinkeby(4).

Initially I did have an implementation that would work locally but having two implementations and having to support two with the local implementation being particularly slow didnt seem worth it. 

This is what pushed me toward the exploration of a way to speed all of this up initially. Should be replaced and unified when we sort out the graph stuff

Reply:
Right, that makes sense